mmtrack/models/trackers/tao_tracker.py

```diff
@@ -0,0 +1,368 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import os
+from collections import defaultdict
+
+import torch
+from mmdet.core import bbox_overlaps
+
+from mmtrack.core import embed_similarity, imshow_tracks
+from ..builder import TRACKERS
+from .base_tracker import BaseTracker
+
+
+@TRACKERS.register_module()
+class TaoTracker(BaseTracker):
```

Comment:
inherit from qdtrack tracker of MOT17

Reply:
These two trackers have the same function but total different implementations (memo / update / track). I think there is no need to inherit from the former tracker.

Reply:
Ok, keep it unchanged.